agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -230,10 +230,13 @@ class backhaul_manager : public btl::transport_socket_thread {
     std::string bssid_bridge_mac;
 
     std::unique_ptr<beerocks::agent_ucc_listener> m_agent_ucc_listener;
-    // vaps map of ruid (key) to VapsList (value)
-    std::unordered_map<sMacAddr, beerocks_message::sVapsList> m_vaps_map;
-    std::unordered_map<sMacAddr, std::vector<beerocks::message::sWifiChannel>>
-        m_radio_capabilities_map;
+
+    struct SRadioInfo {
```

Comment:
Please add doxygen comment on this new structure

Reply:
Fixed